sopel/modules/wikipedia.py

```diff
@@ -8,19 +8,79 @@
 from sopel.module import NOLIMIT, commands, example, rule
 from requests import get
 import re
-
 import sys
+
 if sys.version_info.major < 3:
     from urllib import quote as _quote
     from urlparse import unquote as _unquote
     quote = lambda s: _quote(s.encode('utf-8')).decode('utf-8')
     unquote = lambda s: _unquote(s.encode('utf-8')).decode('utf-8')
+    import HTMLParser
```

Comment:
I think this will fix py2:

```suggestion
    from HTMLParser import HTMLParser
```

But I also haven't tested it, so it might just create a different error :joy_cat:

Reply:
Not testing with python2; guilty as charged. Thanks.